distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -457,23 +461,33 @@ public void testCswIngest() {
 
     @Test
     public void testCswIngestWithMetadataBackup() throws Exception {
+        int startingPostIngestServices = 0;
+        Collection<ServiceReference<PostIngestPlugin>> serviceRefs =
+                getServiceManager().getServiceReferences(PostIngestPlugin.class, null);
+        if (CollectionUtils.isNotEmpty(serviceRefs)) {
+            startingPostIngestServices = serviceRefs.size();
+        }
+        
         getServiceManager().startFeature(true, METACARD_BACKUP_FILE_STORAGE_FEATURE);
-        String fileStorageId = "fileStorageProvider";
         Map<String, Object> storageProps = new HashMap<>();
-        storageProps.put("id", fileStorageId);
-        storageProps.put("outputDirectory", METACARD_BACKUP_DIRECTORY);
-        getServiceManager().createManagedService(
-                "org.codice.ddf.catalog.plugin.metacard.backup.storage.filestorage.MetacardBackupFileStorage",
-                storageProps);
-
-        getServiceManager().startFeature(true, METACARD_BACKUP_PLUGIN_FEATURE);
-        Map<String, Object> properties = new HashMap<>();
-        properties.put("metacardTransformerId", "metadata");
-        properties.put("keepDeletedMetacards", false);
-        properties.put("metacardOutputProviderIds", Collections.singletonList(fileStorageId));
-        getServiceManager().createManagedService(
-                "org.codice.ddf.catalog.plugin.metacard.backup.MetacardBackupPlugin",
-                properties);
+        storageProps.put("outputPathTemplate", METACARD_BACKUP_PATH_TEMPLATE);
+        storageProps.put("metacardTransformerId", "metadata");
+        storageProps.put("keepDeletedMetacards", true);
+        storageProps.put("backupInvalidMetacards", true);
+        getServiceManager().createManagedService("Metacard_File_Storage_Route", storageProps);
+
+        expect("Service to be available: " + MetacardFileStorageRoute.class.getName()).within(30,
+                TimeUnit.SECONDS)
+                .checkEvery(1, TimeUnit.SECONDS)
+                .until(() -> getServiceManager().getServiceReferences(PostIngestPlugin.class, null)
+                        .size(), greaterThan(startingPostIngestServices));
+
+        //Wait for the camel route to come online after the service is created
+        try {
+            TimeUnit.SECONDS.sleep(30);
```

Comment:
Updated the code to get the camel context and verify that the routes are started before continuing.